qemu/tests/block_copy.py

```diff
@@ -46,6 +46,10 @@ def __init__(self, test, params, env, tag):
         self.data_dir = data_dir.get_data_dir()
         self.device = self.get_device()
         self.image_file = self.get_image_file()
+        self.params["monitor_type"] = "qmp"
```

Comment:
need to add it in your configuration file not here, not here. Because VM has really created before call this module.